src/utilities/export.ts

```diff
@@ -0,0 +1,18 @@
+import moment from "moment";
+
+export function getTimestamp(format: string = "YYYY-M-D-H-m-s") {
```

Comment:
using `M` instead of `MM` is a bit problematic, because it means things are no longer easily sortable. For example, `2021-12-1` will be above `2012-8-1`. Rather use the zero-padded alternatives`MM`, `DD`, etc: https://momentjs.com/docs/#/displaying/format/

Reply:
I agree this is better for sorting files. This part is changed.